### PR TITLE
added navlinks to pages

### PR DESCRIPTION
**Pull request by csullivan710**
Added navlinks to header, deleted mockseeds, added button to show page

### CODE REVIEW

File: app/javascript/components/components/Header.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import React, { Component } from 'react'
+import { Card, CardTitle, Col, Button } from 'reactstrap'
```
Comment (hexflores87) [inline app/javascript/components/components/Header.js]: You left in some unused inputs.

Comment (csullivan710) [reply app/javascript/components/components/Header.js]: Will delete unused imports

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -13,7 +15,10 @@ class Header extends Component{
                 <h3>Header</h3>
                 { logged_in && <a href={ sign_out_route }>Sign Out</a> }
                 {!logged_in && <a href={ sign_in_route }>Sign In</a>}
+                <br></br>
+                {this.props.logged_in && <NavLink to="./userrecipes"> Go to your Recipes Page!</NavLink>}
```
Comment (hexflores87) [inline app/javascript/components/components/Header.js]: Noticed you have a dot and not sure if itll work.

File: app/javascript/components/pages/UserRecipes.js [hunk 3]
```diff
@@ -1,11 +1,13 @@
 import React, { Component } from 'react'
+import { Card, CardTitle, Col, Button } from 'reactstrap'
+import { NavLink } from 'react-router-dom'
```
Comment (hexflores87) [inline app/javascript/components/pages/UserRecipes.js:3]: Are these going to be used anytime during the process.

**Review by hexflores87 — APPROVED** [review]

**Pull request merged**